<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="visor">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="btn4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="btn1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="btn2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QPushButton" name="btn3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QPushButton" name="btnSoma">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="2">
         <widget class="QPushButton" name="btn0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QPushButton" name="btnVirg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item row="10" column="4">
         <widget class="QPushButton" name="btnIgual">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="btn5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="btn6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QPushButton" name="btnSub">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="btnBackSpace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="btnDiv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="btnLimpar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="btn7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_17">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_14">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_22">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="btn8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="btn9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="btnMult">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutAvancada">
        <item>
         <widget class="QWidget" name="widgetConv" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="rdbDec">
             <property name="text">
              <string>Dec</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdbOct">
             <property name="text">
              <string>Oct</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdbHex">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdbBin">
             <property name="text">
              <string>Bin</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>46</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExibir">
    <property name="title">
     <string>Exibir</string>
    </property>
    <addaction name="actionPadr_o"/>
    <addaction name="actionCient_fica"/>
   </widget>
   <addaction name="menuExibir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPadr_o">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Padrão</string>
   </property>
  </action>
  <action name="actionCient_fica">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Científica</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>digitoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSoma</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>opClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSub</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>opClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMult</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>opClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDiv</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>opClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIgual</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>igualClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbBin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>convSistemas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbDec</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>convSistemas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbHex</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>convSistemas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbOct</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>convSistemas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>digitoClick()</slot>
  <slot>opClick()</slot>
  <slot>igualClick()</slot>
  <slot>convSistemas()</slot>
 </slots>
</ui>
